ticdc/troubleshoot-ticdc.md

```diff
@@ -275,6 +291,50 @@ If the Old Value feature is not enabled, you cannot tell whether a Row Changed E
 
 For more information, refer to [Open protocol Row Changed Event format](/ticdc/ticdc-open-protocol.md#row-changed-event).
 
+## Does TiCDC support replicating large transactions? Is there any risk?
+
+TiCDC provides partial support for large transactions (more than 5 GB in size). Depending on different scenarios, the following risks might exist:
+
+- When TiCDC's internal processing capacity is insufficient, the replication task error `ErrBufferReachLimit` might occur.
+- When TiCDC's internal processing capacity is insufficient or the throughput capacity of TiCDC's downstream is insufficient, out of memory (OOM) might occur.
+
+If you encounter errors above, it is recommended to use BR for incrementally restoring data in large transactions. The detailed operations are as follows:
+
+1. Record the `checkpoint-ts` of the changefeed that is terminated due to large transactions, use this TSO as the `--lastbackupts` of the BR incremental backup, and execute [Back up incremental data](/br/backup-and-restore-tool.md#back-up-incremental-data).
+2. After backing up the incremental data, you can find a log similar to `["Full backup Failed summary : total backup ranges: 0, total success: 0, total failed: 0"] [BackupTS=421758868510212097]` in the BR log output. Record the `BackupTS` in this log.
+3. [Restore the incremental data](/br/backup-and-restore-tool.md#restore-incremental-data).
+4. Create a new changefeed and start the replication task from `BackupTS`.
+5. Delete the old changefeed.
+
+## When the downstream of a changefeed is a database similar to MySQL and TiCDC executes a long DDL statement, all other changefeeds are blocked. How should I handle the issue?
+
+1. Firstly, suspend the execution of the changefeed of the long DDL. At this point, you can find that after you suspend this changefeed, other changefeeds are no longer blocked.
+2. Search for the `apply job` field in the TiCDC log and confirm the time-consuming DDL `StartTs`.
+3. Execute the DDL statement downstream manually, and perform the following operations after execution.
+4. Modify the changefeed configuration and add the above `StartTs` to the `ignore-txn-start-ts` configuration item.
+5. Restore a suspended changefeed.
+
+## After I upgrade TiCDC cluster to v4.0.8, changefeed reports the error `[CDC:ErrKafkaInvalidConfig]Canal requires old value to be enabled`
+
+Since v4.0.8, if changefeed uses canal or canal-json protocol output, TiCDC checks whether you enable the Old Value feature. If you disable it, an error is reported. You can solve the problem by following the steps below:
```

Comment:
```suggestion
Since v4.0.8, if the `canal` or `canal-json` protocol is used for output in a changefeed, TiCDC checks whether the old value feature is also enabled. If you disable the old value feature, this error is reported. To fix the error, take the following steps:
```